pdns/dnsdist-lua-rules.cc

```diff
@@ -915,24 +915,39 @@ void parseRuleParams(boost::optional<luaruleparams_t> params, boost::uuids::uuid
   uuid = makeRuleID(uuidStr);
 }
 
+typedef std::unordered_map<std::string, boost::variant<bool, int, std::string, std::vector<std::pair<int,int> > > > localbind_t;
```

Comment:
`localbind_t` is not a really good name and we already use it in another unit, perhaps `ruleparams_t` or something like that?

Reply:
Sure.